spec/models/instance/settings_spec.rb

```diff
@@ -1,45 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe Instance::Settings, type: :model do
-  ALL_COMPONENTS = [:A, :B, :C, :D, :E, :F]
-  ENABLED_COMPONENTS = [:A, :C, :E]
+  ALL_COMPONENTS = [:A, :B, :C, :D, :E, :F].freeze
```

Comment:
@lowjoel After played with this PR for a while, I think we should put a `self` before constant, so `self::ALL_COMPONENTS`, otherwise these contacts are global and can be accessed in other places. what do you think ?